### ISSUE TITLE
Logger with multiple tags

### ISSUE THREAD

**Issue by MartinAnon** [tinylog-org/tinylog#201]
Hi there,

I was wondering if there is a way to assign multiple tags to a message. As far as the documentation is concerned, I read that you can either use a TaggedLoger or Logger.tag(...) for a single tag. 

**Example:**
A java program is currently configured to log to console and into a file using the following **tinylog.configuration**:
```
writerConsole          = console
writerConsole.tag      = CONSOLE, JAVAPR@info
writerConsole.format   = ....

writerJavaProgram      = rolling file
writerJavaProgram.tag  = JAVAPR
...
```

Now we want to log specific information in specific files, f.e. monitor all user logins in a specific logging file, that only contains that information. 
```
writerLogins = rolling file
writerLogins.tag = LOGINS
...
```

Of course you could now use 2 separate TaggedLoggers, one to log to console+file and the other to log into the logins file.
```
public static final String JAVAPR_TAG = "JAVAPR", LOGINS_TAG = "LOGINS";
TaggedLogger javaprLogger = Logger.tag(JAVAPR_TAG), loginsLogger = Logger.tag(LOGINS_TAG);
//...
javaPrLogger.info("Started program ...");
//...
String userLoginMessage = "User abc logged in...";
javaprLogger.info(userLoginMessage);
loginsLogger.info(userLoginMessage);
```

But is there maybe an easier way to do that without calling .info() multiple times that I'm missing ?

Thanks in Advance!

**Comment by pmwmedia** [comment]
Currently, tinylog 2 supports only one tag per log entry. I don't know any better workaround than the one you have already figured out.  Therefore, I have converted your question into a feature request. I think this would a great enhancement also for other tinylog users. I'm open for pull requests.

**Comment by pmwmedia** [comment]
Implemented by pull request #213
